gradle.properties

```diff
@@ -6,14 +6,14 @@
 # http://www.gradle.org/docs/current/userguide/build_environment.html
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8 -Dkotlin.daemon.jvm.options=--illegal-access=permit
```

Comment:
out of curiosity what's the error

Reply:
oh i didnt notice that. it mustve been automatically added by something so idk

Reply:
remoooove